revvy/revvy_utils.py

```diff
@@ -70,14 +69,27 @@ def __init__(self, robot: Robot, sw_version, revvy_ble):
 
         self._scripts = ScriptManager(self)
         self._background_controlled_scripts = ScriptManager(self)
-        self._autonomous = 0;
+        self._autonomous = 0
         self._config = empty_robot_config
 
         self._status_code = RevvyStatusCode.OK
         self.exited = Event()
 
         self.start_remote_controller = self._remote_controller_thread.start
 
+    def process_autonomous_requests(self):
```

Comment:
I guess this is fine because this is preexisting, but having a setter, a getter, and a control function in a different class for each request type is a bit wasteful. I'd probably pass the request, or the request type enum value directly to `_background_controlled_scripts` and match on the value there. This way it's harder to forget to update a place of use if something changes.

Reply:
I am just following API paradigm here. Like values are internal to class and how they decide true or false is a blackbox.